debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/TopicSelector.java

```diff
@@ -60,6 +60,17 @@ public String getTopic(String databaseName, String tableName) {
                 return String.join(delimiter, prefix, databaseName, tableName);
             }
 
+            /**
+             * Get the name of the heartbeat topic for the given server. This method returns
+             * "{@code <prefix>-heartbeat}".
+             * 
+             * @return the topic name; never null
+             */
+            @Override
+            public String getHeartbeatTopic() {
```

Comment:
How about making this topic configurable, defaulting to something like "__debezium-hearbeat-<server>"?

Reply:
Done, the naming pattern is now <hearbeat_prefix>.<server_prefix>